test/s1/appframework/appframework_test.go

```diff
@@ -1061,12 +1064,44 @@ var _ = Describe("s1appfw test", func() {
 			   * Verify App enabled  and version by running splunk cmd
 			*/
 
-			//################## SETUP ####################
+			// ################## SETUP FOR MONITORING CONSOLE ####################
```

Comment:
Original comment is matching the block comment above better to leave it as it is. 
i.e. `//################## SETUP ####################`